src/Analyzer.Core.BuiltInRuleTests/Tests/TA-000029/TA-000029.json

```diff
@@ -148,4 +148,4 @@
 
     ]
   }
-]
\ No newline at end of file
+]
```

Comment:
Revert this change please.

Reply:
The whole added section of the template?

Reply:
This file only has a single change, which is just a newline at the end.  Since there's no meaningful change, it's best to revert it and remove the change altogether.

Reply:
Ooh, I couldn't tell that that was a new line change. 
I believe I have gotten rid of it now